nucypher/network/middleware.py

```diff
@@ -36,47 +37,96 @@ class NotFound(UnexpectedResponse):
     pass
 
 
-class RestMiddleware:
-    log = Logger()
+class NucypherMiddlewareClient:
+    library = requests
 
-    class _Client:
+    @staticmethod
+    def response_cleaner(response):
+        return response
```

Comment:
Why implement this? It doesn't do anything as far as I can see?

Reply:
Yeah, it's *this close* to being an `@abstractmethod`, but it does get called.  In the "real" middleware, it doesn't need to do anything at this moment.